icons/poi_school.svg

```diff
@@ -1,5 +1,5 @@
 <svg width="20" height="20" xmlns="http://www.w3.org/2000/svg">
  <g style="display:inline;stroke:#000;stroke-opacity:0">
-  <path style="display:inline;fill:#000;fill-opacity:1;stroke:#fff;stroke-width:2;stroke-linecap:round;stroke-linejoin:round;stroke-dasharray:none;stroke-opacity:1;paint-order:stroke markers fill" d="M 11,1 3,4 9,6 Z M 5,9 V 19 H 15 V 9 Z M 9,6 v 3 h 2 V 1 Z"/>
+  <path style="display:inline;fill:#000;fill-opacity:1;stroke:#fff;stroke-width:2;stroke-linecap:round;stroke-linejoin:round;stroke-dasharray:none;stroke-opacity:1;paint-order:stroke markers fill" d="m 11,1 -6.999801,3 6,2 z M 5,9 V 19 H 15 V 9 Z m 5.000199,-3 0,3 H 11 V 1 Z" transform="matrix(-1 0 0 1 20 0)"/>
```

Comment:
I wonder if the flagpole could be slightly taller. The square kind of dominates the icon, especially when other POI icons are nearby.

Reply:
Here I tried shrinking the building square from 10x10 to 9x9 to make the flagpole proportionally taller.

<img width="407" alt="Screen Shot 2023-04-10 at 7 16 55 PM" src="https://user-images.githubusercontent.com/58491489/231038311-c9789c43-74dd-4f7f-94ae-69b890e94a05.png">
<img width="269" alt="Screen Shot 2023-04-10 at 7 16 17 PM" src="https://user-images.githubusercontent.com/58491489/231038313-3571bcab-dfb1-4344-b3f0-0432e2864fe9.png">